Lab/demo/Lab/Plugins/PCA/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ cgal_lab_plugin(create_obb_mesh_plugin Create_obb_mesh_plugin)
 target_link_libraries(create_obb_mesh_plugin PUBLIC scene_surface_mesh_item
                                                     scene_polygon_soup_item
                                                     scene_selection_item
-                                                    scene_points_with_normal_item)
+                                                    scene_points_with_normal_item CGAL::Eigen3_support)
```

Comment:
Actually, it should be inherited from the items, unless the code of `scene_points_with_normal` itself uses Eigen.

Reply:
It only inherits if any of the dependency declares its Eigen dependency public I guess. Note that `create_obb_mesh_plugin.cpp`   used Eigen directly.